torch/export/_unlift.py

```diff
@@ -729,14 +729,35 @@ def _unlift_exported_program_lifted_states(
     graph = unlift_gm.graph
     placeholders = graph.find_nodes(op="placeholder")
     if check_guards and placeholders and ep.example_inputs:
+        sig = inspect.signature(unlift_gm.forward)
         input_paths = _get_input_paths(
             ep.example_inputs,
-            inspect.signature(unlift_gm.forward),
+            sig,
         )
+
+        # TODO (tmanlaibaatar)
+        # This is band-aid solution to export new tracer replacing
```

Comment:
cc: @avikchaudhuri  I guess we need to fix this before Autoparallel needing to run the guards in a different way or if they use lower level export APIs.